care/facility/tests/test_asset_location_api.py

```diff
@@ -78,3 +82,9 @@ def test_create_asset_location_invalid_middleware(self):
         self.assertEqual(
             response.data["middleware_address"][0].code, "invalid_domain_name"
         )
+
+    def test_delete_asset_location(self):
+        response = self.client.delete(
+            f"/api/v1/facility/{self.facility.external_id}/asset_location/{self.asset_location.external_id}/",
+        )
+        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
```

Comment:
@Pranshu1902  Kindly read the issue correctly. This test should assert that the API rejects deletion of such locations.

<img width="946" alt="image" src="https://github.com/coronasafe/care/assets/25143503/86d462e6-f917-4b82-a060-0db800e6a03b">